sample/build.gradle.kts

```diff
@@ -55,10 +58,19 @@ dependencies {
   implementation(libs.dagger)
 
   testImplementation(libs.androidx.compose.ui.testing.junit)
-  testImplementation(libs.coroutines.test)
   testImplementation(libs.junit)
+  testImplementation(libs.kotlin.coroutines.test)
   testImplementation(libs.molecule.runtime)
   testImplementation(libs.truth)
   testImplementation(libs.turbine)
   testImplementation(libs.robolectric)
+
+  androidTestImplementation(libs.androidx.test.core)
+  androidTestImplementation(libs.androidx.test.rules)
+  androidTestImplementation(libs.androidx.test.runner)
+  debugImplementation(libs.androidx.compose.ui.testing.manifest)
```

Comment:
Why's this one debugImpl vs the others?

Reply:
Apparently it adds some properties to the debug manifest necessary for testing: https://developer.android.com/jetpack/compose/testing#setup. I tried using `androidTestImplementation` instead but get an error.

I googled and found this:
> You can override or remove completely the android:launchMode attribute by [node markers](https://developer.android.com/studio/build/manifest-merge#node_markers).
By default, androidTest runs in the debug build type. So adding a proper node marker to Manifest in the/debug directory will override it for the app used by androidTest tests.